portfolio/src/main/webapp/mickey.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Mickey | Peter Geertsema's Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="nav-bar">
+      <ul id="nav-list">
+        <li class="nav-item"><a class="nav-inactive" href="index.html">Home</a></li>
+        <li class="nav-item"><a class="nav-active" href="mickey.html">Mickey</a></li>
+        <li class="nav-item"><a class="nav-inactive" href="projects.html">Projects</a></li>
+      </ul>
+    </div>
+    <div id="content">
+      <h1>Here are some pictures of Mickey!</h1>
+      <p>I always sent these around to class groupchats in high school before exams :)</p>
+      <img class="mickey-media" src="images/Mickey Outside.jpg"/>
```

Comment:
just wanted to confirm, does image show up clearly with the space in the file name?

Reply:
Yes, the images show up normally. 
Should I rename them to use dashes or underscores for backwards compatibility?

Reply:
yes please, that'd be great!

Reply:
Done!